app/assets/stylesheets/_settings.scss

```diff
@@ -57,9 +57,11 @@ $medium-gray: #cacaca;
 $dark-gray: #8a8a8a;
 $black: #0a0a0a;
 $white: #fefefe;
-$body-background: $white;
+$body-background: #fffaf0;
 $body-font-color: $black;
-$body-font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;
+$font-family: 'Source Sans Pro', sans-serif;
+$title-font-family: 'Satisfy', cursive;
+$body-font-family: $font_family;
```

Comment:
Name of variable `font_family` should be written in all lowercase letters with hyphens instead of underscores